Converted to python3 compatible with 2to3.4 -wn <dir>

els-pnw opened the pull request:
Had to move to python3 for Re to work in Limnoria's latest build, therefore I had to convert 2to3.4 against supybot-jira.

els-pnw committed:
Converted to python3 compatible with 2to3.4 -wn <dir>

rm-you commented:
I realize a lot of this code is really ugly, and it's not necessarily yours, sorry. T_T
It could use quite a bit of refactoring, and it may not be a fair request, but I was hoping you could fix the lines you're working on at least (since you're already touching them and it's simple changes).

els-pnw committed:
a little code clean up.  Mostly changes to
if (( foo in bar) != True) to if (foo not in bar)

els-pnw committed:
url = ''.join((self.server, '/browse/', issue.key)) statements added an extra forward slash.
https://issues.jboss.org//browse/MSGQE-17
This had a negative user experience with some irc clients / browsers when clicked.  %s/\/browse\//browse\//g

rm-you commented:
Thanks for fixing those! :)
